api-contracts/openapi/paths/v2/workflow-runs/workflow_run.yaml

```diff
@@ -90,3 +90,112 @@ listWorkflowRuns:
     summary: List workflow runs
     tags:
       - Workflow Runs
+
+
+listTaskEventsForWorkflowRun:
+  get:
+    x-resources: ["tenant"]
```

Comment:
Since the workflow version ID uniquely identifies a tenant, it's not necessary to put the `tenant` in the path. You can change this `x-resources` to `["tenant", "workflow-version"]` and write a populator for the workflow version which returns the tenantId as the `uniqueParentId`. 

We're not fully consistent in this approach but it's definitely preferred -- this gets around a class of security issues where the workflow version id corresponds to a different tenant id than what's been passed in. It also makes the URLs cleaner and reduces the number of required parameters.

Reply:
wait, just to make sure I'm following - we have an existing populator for `workflow-run`. Will that work here?

Reply:
I thought we wanted to populate on `workflow-version`? The `workflow-run` populator won't work without modifications for v2 queries.

Reply:
It might be helpful to [take a look at this implementation](https://github.com/hatchet-dev/hatchet-v2-experimental/blob/feat--task-events-views/api/v1/server/handlers/v2/workflow-runs/get.go#L14-L71) to see if it makes sense to you. I was thinking we'd pass the workflow run since we have that in the path on the frontend, and then we'd get the workflow version from it. lmk if that's not how you were thinking this should work though!

Reply:
Implementation-wise the handler itself looks fine, but we need a `v2-workflow-run` populator and shouldn't need both tenant and workflow-run in the path params.

Reply:
cool, sounds good! I'll add that

Reply:
pushed a couple changes - let me know if that looks right!